DAL/Repositories/UnitOfWork.cs

```diff
@@ -94,7 +98,55 @@ public BaseRepository<Topic> Topics
                 return topicRep;
             }
         }
-        
+        public BaseRepository<Specialization> SpecializationRep
```

Comment:
There are no postfix **Rep** in other repositories. I suppose need delete this postfix.